.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
```

Comment:
Is there a way to add the sdist and wheel to the draft? It is not strictly necessary to publish on GitHub but I think it's "nice to have". The good thing is that you could validate it before pushing the button.

Reply:
Ehhh I can check

Reply:
Don't spend too much time on it.

Reply:
Yes it is possible:

See: https://github.com/ncipollo/release-action#action-inputs

Any other options to include?

Reply:
Cool, just add sdist and wheel I'd say. Then we can make the publish script just run twine on the files attached to the release?

Reply:
How does one do that last part? is that something that twine does out of the box?

Reply:
No, I since you trigger it on "released" I just figured you had access to the release metadata (files included) but maybe that's not the case? I think twine just needs the sdist and wheel, I don't think it needs the setup.py.

Reply:
I think you should be able to access the asset_url and download that using curl and unzip it: https://developer.github.com/v3/activity/events/types/#releaseevent.

Reply:
😴 I think we can leave it as it is then. 

They will always be the same, since the release is made from the tag and tags is what you would check out anyway. Also reuploading a tag would also rewrrite the assets, so I see no gain on that workflow.

Reply:
I see, makes sense.